tests/e2e/auth/client/testutil/cli_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"testing"
```

Comment:
nit: cant comment high enough but could we replace the build flag from norace to e2e

I can jump in to clean up ci after this pr Is merged

Reply:
on second thought, ignore this comment and ill pick it up after this is merged

Reply:
Putting automerge then